notus/scanner/loader/json.py

```diff
@@ -100,12 +103,21 @@ def load_package_advisories(
             for package_dict in fixed_packages:
                 full_name = package_dict.get("full_name")
                 if full_name:
-                    package = RPMPackage.from_full_name(full_name)
+                    if operating_system.find("debian") != -1:
+                        package = DEBPackage.from_full_name(full_name)
+                    else:
+                        package = RPMPackage.from_full_name(full_name)
                 else:
-                    package = RPMPackage.from_name_and_full_version(
-                        package_dict.get("name"),
-                        package_dict.get("full_version"),
-                    )
+                    if operating_system.find("debian") != -1:
```

Comment:
I guess this should be
```suggestion
                    if "debian" in operating_system:
```